documentation/hang/step2/using_inspector.md

```diff
@@ -1 +1,54 @@
-// TODO
+# Using Inspectors
+
+Inspectors are very useful on development. It allow us to step into our
+programs and get our hands on every slice of our code and values of its
+local variables.
+
+## Attach to the Running Process
+
+Normally, Node.js process will not open inspector port for listening
+inspector client connections. We can send a signal `SIGUSR1` to the running
+process and then the inspector will start listening for new debugger
+connections.
```

Comment:
Worth mentioning that you can start the process with a specific port open. This should probably come first, leaving the SIGUSR1 as a secondary approach (especially since it doesn't work on Windows)